### DISCUSSION TITLE
connectome pipeline output

### DISCUSSION THREAD

**Discussion by ParkvilleGeek** [PennLINC/qsiprep#328] [Q&A]
hello
We have been running the qsiprep connectome pipeline and we would like some help interpreting the output - the .mat files and images.  eg are values raw counts?

We have been looking for online resources/guides but there does not appear to be any info about what files are produced by the connectome module.

Many thanks

**Comment by mattcieslak** [comment]
Thanks for bringing this up. The reconstruction pipelines definitely need some extra documentation, but the values in the mat files are all from the original software. Which pipeline are you using?
